docs/CONFIG-PROPERTIES.md

```diff
@@ -44,6 +44,10 @@
 | newlog.gzipfiles.ondisk.maxmegabytes | integer in Mbytes | 2048 | the quota for keepig newlog gzip files on device |
 | process.cloud-init.multipart | boolean | false | help VMs which do not handle mime multi-part themselves |
 | edgeview.authen.jwt | edgeview session jwt token | empty string(edgeview disabled) | format as standard JWT for websocket session for temporary testing, this configitem will be removed once controllers are setup to send EdgeViewConfig in configuration |
+| netdump.enable | boolean | true | enable publishing of network diagnostics (as tgz archives to /persist/netdump) |
```

Comment:
Btw. there is no "netdump traffic" right now - we do not upload netdump archives to the controller, just store them in the /persist partition.

Reply:
Okay, got it. Question was more to netdump traffic sending to controller from EVE as this thing can get big and we need to have some kind of mechanism to handle this. But since it's stored on persist partition its alright. Maybe it'll be good idea to rotate netdump or archive it so that we can't fill persist partition

Reply:
Yes it is rotated, see the `netdump.topic.maxcount` config property below.